openflow13/nx_action.go

```diff
@@ -897,3 +897,60 @@ func (a *NXActionOutputReg2) UnmarshalBinary(data []byte) error {
 	a.MaxLen = binary.BigEndian.Uint16(data[n:])
 	return err
 }
+
+type NXActionDecTTL struct {
+	*NXActionHeader
+	controllers uint16  // number of controller
+	zeros       []uint8 // 4 byte with zeros
+}
```

Comment:
Maybe this is a misinterpretation of NXAST_DEC_TTL_CNT_IDS. This actions has variable length: the fixed data above is followed by a sequence of controller IDs, the number specified by 'controllers' in the fixed data. I don't see anything about variable-length data in this implementation.

Reply:
I add new struct specific for NXAST_DEC_TTL_CNT_IDS, and add a field for controller IDs.